src/main/java/org/springframework/data/elasticsearch/annotations/Score.java

```diff
@@ -0,0 +1,23 @@
+package org.springframework.data.elasticsearch.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.data.annotation.ReadOnlyProperty;
+
+/**
+ * Specifies that this field is used for storing the document score.
+ * 
+ * @author Sascha Woo
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+@Documented
+@Inherited
+@ReadOnlyProperty
```

Comment:
I've just checked the `EntityWriter` implementation and it looks like it

Reply:
I've just filed [DATAES-464](https://jira.spring.io/browse/DATAES-464) and am going to push a fix for that. With that in place, the read-only properties shouldn't be written to Elasticsearch.

Reply:
Nice, will rebase the PR tomorrow and address the other stuff as well.